test/__mocks__/get.company.psc.mock.ts

```diff
@@ -79,6 +79,51 @@ export const MOCK_GET_COMPANY_PSC_RESOURCE = {
   "total_results": ''
 };
 
+export const MOCK_GET_COMPANY_PSC_RESOURCE_FOR_GOV = {
+  "active_count": '1',
+  "ceased_count": '',
+  "items": [
+    {
+      "natures_of_control": [
+        'ownership-of-shares-more-than-25-percent-registered-overseas-entity',
+        'ownership-of-shares-more-than-25-percent-as-trust-registered-overseas-entity',
+      ],
+      "kind": "legal-person-with-significant-control",
+      "name_elements": {
+        "middleName": "Notreal",
+        "forename": "Random",
+        "title": "Mr",
+        "surname": "Person"
+      },
+      "name": "Mr Random Notreal Person",
+      "notified_on": "2022-04-06",
+      "nationality": "British",
+      "address": {
+        "postal_code": "CF14 3UZ",
+        "premises": "Companies House",
+        "locality": "Limavady",
+        "country": "Wales",
+        "address_line_1": "Crown Way"
+      },
+      "country_of_residence": "Wales",
+      "date_of_birth": {
+        "month": '8',
+        "year": '1983'
+      },
+      "links": {
+        "self": "/company/OE111129/persons-with-significant-control/individual/RandomeaP1EB70SSD9SLmiK5Y"
```

Comment:
I'm not sure this matters much but the /individual/ in this link would be /legal-person/